archetypes/multilingual/interfaces.py

```diff
@@ -1,5 +1,10 @@
 # -*- coding: utf-8 -*-
 from plone.app.multilingual.interfaces import ITranslatable
+from zope.interface import Interface
+
+
+class IArchetypesMultilingualInstalled(Interface):
```

Comment:
you may want to inherit from archetypes.multilingual.interfaces.IArchetypesMultilingualInstalled
this way this layer is always more specific than the one from p.a.m